daemon/dpudaemon.go

```diff
@@ -6,7 +6,9 @@ import (
 	"net"
 
 	"github.com/go-logr/logr"
+	deviceplugin "github.com/openshift/dpu-operator/daemon/device-plugin"
 	"github.com/openshift/dpu-operator/daemon/plugin"
+	pb2 "github.com/openshift/dpu-operator/tree/main/dpu-api/gen"
```

Comment:
These imports are invalid - as were required for local testing.
once PR is merged, a second PR will be required to correct the imports